### ISSUE TITLE
Cookie AssertionError when loading cookie with a username with uppercase

### ISSUE THREAD

**Issue by Rayanox** [ohld/igbot#1028]
## Please follow the guide below

- Issues submitted without this template format will be **ignored**.
- You will be asked some questions and requested to provide some information, please read them **carefully** and answer completely.
- Put an `x` into all the boxes [ ] relevant to your issue (like so [x]).
- Use the *Preview* tab to see how your issue will actually look like.

---

### Before submitting an issue, make sure you have:
- [X ] Updated to the latest version
- [X ] Read the [README](https://github.com/instagrambot/instabot/blob/master/README.md) and [docs](https://github.com/instagrambot/instabot/tree/master/docs)
- [X ] [Searched](https://github.com/instagrambot/instabot/search?type=Issues) the bugtracker for similar issues including **closed** ones

### Purpose of your issue?
- [X ] Bug report (encountered problems/errors)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections requests more details for particular types of issues, you can remove any section (the contents between the triple ---) not applicable to your issue.

---

### For a *bug report*, you **must** include the Python version used, *code* that will reproduce the error, and the *error log/traceback*.

Paste the output of ``python -V`` here:

Code:

```python
"""
    instabot example
    Workflow:
        Follow user's followers by username.
"""

import argparse
import os
import sys
import json

sys.path.append(os.path.join(sys.path[0], '../'))
from instabot import Bot

parser = argparse.ArgumentParser(add_help=True)
parser.add_argument('-u', type=str, help="username")
parser.add_argument('-p', type=str, help="password")
parser.add_argument('-proxy', type=str, help="proxy")
parser.add_argument('-usernameTarget', type=str, help='usernameTarget')
args = parser.parse_args()

bot = Bot()
bot.login(username=args.u, password=args.p,
          proxy=args.proxy, use_cookie=True)

result = bot.convert_to_user_id("Rayanox")
print("result = " + result);

exit(0)
```

Error/Debug Log:

```python
PS C:\Users\rbenhmidane\Documents\Repositories\Instagram-follower-booster\Scripts> python .\test.py -u "Rayanox92" -p "*****" -usernameTarget "Rayanox92"
2019-09-03 11:37:44,748 - INFO - Instabot Started
cookie_username = rayanox92
self.username = Rayanox92
Traceback (most recent call last):
  File ".\test.py", line 28, in <module>
    proxy=args.proxy, use_cookie=True)
  File "C:\Users\rbenhmidane\AppData\Local\Programs\Python\Python37\lib\site-packages\instabot\bot\bot.py", line 369, in login
    if self.api.login(**args) is False:
  File "C:\Users\rbenhmidane\AppData\Local\Programs\Python\Python37\lib\site-packages\instabot\api\api.py", line 198, in login
    self.load_uuid_and_cookie(load_cookie=use_cookie, load_uuid=use_uuid)
  File "C:\Users\rbenhmidane\AppData\Local\Programs\Python\Python37\lib\site-packages\instabot\api\api.py", line 159, in load_uuid_and_cookie
    return load_uuid_and_cookie(self, load_uuid=load_uuid, load_cookie=load_cookie)
  File "C:\Users\rbenhmidane\AppData\Local\Programs\Python\Python37\lib\site-packages\instabot\api\api_login.py", line 202, in load_uuid_and_cookie
    assert cookie_username == self.username
AssertionError
```

I added two custom logs to show the value of 'cookie_username' and 'self.username' just  before the assertion.

---

### Describe your issue

When using an account name that contains an "uppercase", the cookie loading fails and throws an assertion error (api_login.py:200). I have just investigate on this issue, using the last version of the scripts, and I found that the username cookie loaded is stored in lowercase, and when checking that it equals my username it fails because there is an uppercase in my  username. A simple fix is to force the username to be lower just for the comparison, it's not the best solution but it's still better than the existing code.

**Comment by MertisWee** [comment]
sooo, what do i need to do ?

**Comment by Rayanox** [comment]
Nothing. It has already been fixed -> https://github.com/instagrambot/instabot/pull/1029

**Comment by Abdulsamipy** [comment]
> Nothing. It has already been fixed -> #1029

pardon me .. can you tell me what to do to fix the issue?
